src/main/groovy/com/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -181,6 +182,10 @@ class MarkLogicPlugin implements Plugin<Project> {
         project.task("mlWatch", type: WatchTask, group: modulesGroup, description: "Run a loop that checks for new/modified modules every second and loads any that it finds")
         project.task("mlDeleteModuleTimestampsFile", type: DeleteModuleTimestampsFileTask, group: modulesGroup, description: "Delete the properties file in the build directory that keeps track of when each module was last loaded")
 
+        String schemasDataGroup = "ml-gradle Schemas"
+        project.task("mlLoadSchemasData", type: LoadSchemasDataTask, group: schemasDataGroup, description: "Loads special-purpose data into the schemas database (XSD schemas, Inference rules, and [MarkLogic 9] Extraction Templates)")
+        project.task("mlReloadSchemasData", type: LoadSchemasDataTask, dependsOn: ["mlClearSchemasDatabase", "mlLoadSchemasData"], group: schemasDataGroup, description: "Loads special-purpose data into the schemas database (XSD schemas, Inference rules, and [MarkLogic 9] Extraction Templates)")
```

Comment:
mlClearSchemasDatabase -- a pre-existing task that you wrote :)

Your interpretation of SchemasData is correct -- lots of things can go in the schemas database.  But I accept your reasoning too, and will redo the PRs without 'Data'